src/backend/access/common/reloptions.c

```diff
@@ -515,22 +515,20 @@ default_reloptions(Datum reloptions, bool validate, char relkind,
 
 		compresstype = values[5];
 
```

Comment:
The fix should remove quicklz from valid compression type list. It is better to add comment here we reserve quicklz in catalog for ease of upgrade.